### DISCUSSION TITLE
Request failed with status code 429

### DISCUSSION THREAD

**Discussion by gsutton57** [FlowiseAI/Flowise#435] [Q&A]
I am using the simple chatGPT template and when I run it, it takes a while and then gives me this error code:
Request failed with status code 429 and body {"error":{"message":"You exceeded your current quota, please check your plan and billing details.","type":"insufficient_quota","param":null,"code":null}}

Any help would be appreciated.  I am a beginner, sorry.

**Comment by trepont** [comment]
Please verify the token limit in your ChatGPT account, as it appears to have exceeded the allocated limit. Kindly consider using alternative login credentials and upgrading your account accordingly

**Comment by chungyau97** [comment]
Hi @gsutton57,

Sorry for the late reply.

Troubleshooting steps:
1. [Open AI](https://platform.openai.com/): Personal > Manage account > Usage
2. Check status of Free Trial usage
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/7176eb15-1d54-4baf-ac42-c4213821faa5)

Expired and insufficient funds will cause this error.

Furthermore do come join our [discord](https://discord.gg/AfZaQA5k); You will find this solution in the support channel.

**Reply by rmminusrslash** [reply]
I had the same error. Beginners or non-developers don't understand them. I would say consider catching it and resurfacing a more meaningful error message with instructions or a link to a help page?

**Reply by gsutton57** [reply]
Thanks, I will give it a try!
